src/Microsoft.AspNetCore.HttpsPolicy/HttpsRedirectionMiddleware.cs

```diff
@@ -127,16 +143,34 @@ private void CheckForHttpsPorts()
                     // If we find multiple different https ports specified, throw
                     if (httpsPort.HasValue && httpsPort != bindingAddress.Port)
                     {
-                        throw new ArgumentException("Cannot determine the https port from IServerAddressesFeature, multiple values were found. " +
+                        _logger.LogWarning("Cannot determine the https port from IServerAddressesFeature, multiple values were found. " +
```

Comment:
I still would throw here. This isn't occurs only when multiple https ports were set.

Reply:
They wanted the site to run, above and beyond any other requirement.

Reply:
True, this isn't a startup error, only first request.